File: clients/fireback-ios/firebackios/modules/workspaces/WorkspaceInviteEntity.swift
class WorkspaceInviteEntity : Codable {
    private var _passportMode: String?
    var `PassportMode`: String? {
        set { _passportMode = newValue }
        get { return _passportMode }
    }
    private var _coverLetter: String?
    var `CoverLetter`: String? {
        set { _coverLetter = newValue }
        get { return _coverLetter }
    }
    private var _targetUserLocale: String?
    var `TargetUserLocale`: String? {
        set { _targetUserLocale = newValue }
        get { return _targetUserLocale }
    }
    private var _email: String?
    var `Email`: String? {
        set { _email = newValue }
        get { return _email }
    }
    private var _workspace: WorkspaceEntity?
    var `Workspace`: WorkspaceEntity? {
        set { _workspace = newValue }
        get { return _workspace }
    }
    private var _firstName: String?
    var `FirstName`: String? {
        set { _firstName = newValue }
        get { return _firstName }
    }
    private var _lastName: String?
    var `LastName`: String? {
        set { _lastName = newValue }
        get { return _lastName }
    }
    private var _inviteeUserId: String?
    var `InviteeUserId`: String? {
        set { _inviteeUserId = newValue }
        get { return _inviteeUserId }
    }
    private var _phoneNumber: String?
    var `PhoneNumber`: String? {
        set { _phoneNumber = newValue }
        get { return _phoneNumber }
    }
    private var _used: Bool?
    var `Used`: Bool? {
        set { _used = newValue }
        get { return _used }
    }
    private var _role: RoleEntity?
    var `Role`: RoleEntity? {
        set { _role = newValue }
        get { return _role }
    }
}

File: clients/fireback-react/src/sdk/fireback/modules/workspaces/WorkspaceInviteEntity.ts
import {
    BaseDto,
    BaseEntity,
} from "../../core/definitions"
import {
    RoleEntity,
} from "./RoleEntity"
import {
    WorkspaceEntity,
} from "./WorkspaceEntity"
// In this section we have sub entities related to this object
// Class body
export type WorkspaceInviteEntityKeys =
  keyof typeof WorkspaceInviteEntity.Fields;
export class WorkspaceInviteEntity extends BaseEntity {
  public coverLetter?: string | null;
  public targetUserLocale?: string | null;
  public value?: string | null;
  public workspace?: WorkspaceEntity | null;
  public firstName?: string | null;
  public lastName?: string | null;
  public used?: boolean | null;
  public role?: RoleEntity | null;
      roleId?: string | null;
  public static Navigation = {
      edit(uniqueId: string, locale?: string) {
          return `${locale ? '/' + locale : ''}/workspace-invite/edit/${uniqueId}`;
      },
      create(locale?: string) {
          return `${locale ? '/' + locale : ''}/workspace-invite/new`;
      },
      single(uniqueId: string, locale?: string) {
          return `${locale ? '/' + locale : ''}/workspace-invite/${uniqueId}`;
      },
      query(params: any = {}, locale?: string) {
          return `${locale ? '/' + locale : ''}/workspace-invites`;
      },
      /**
      * Use R series while building router in CRA or nextjs, or react navigation for react Native
      * Might be useful in Angular as well.
      **/
      Redit: "workspace-invite/edit/:uniqueId",
      Rcreate: "workspace-invite/new",
      Rsingle: "workspace-invite/:uniqueId",
      Rquery: "workspace-invites",
  };
  public static definition = {
  "name": "workspaceInvite",
  "http": {},
  "gormMap": {},
  "fields": [
    {
      "name": "coverLetter",
      "type": "string",
      "computedType": "string",
      "gormMap": {}
    },
    {
      "name": "targetUserLocale",
      "type": "string",
      "computedType": "string",
      "gormMap": {}
    },
    {
      "name": "value",
      "type": "string",
      "validate": "required",
      "computedType": "string",
      "gormMap": {}
    },
    {
      "name": "workspace",
      "type": "one",
      "target": "WorkspaceEntity",
      "validate": "required",
      "computedType": "WorkspaceEntity",
      "gormMap": {}
    },
    {
      "name": "firstName",
      "type": "string",
      "validate": "required",
      "computedType": "string",
      "gormMap": {}
    },
    {
      "name": "lastName",
      "type": "string",
      "validate": "required",
      "computedType": "string",
      "gormMap": {}
    },
    {
      "name": "used",
      "type": "bool",
      "computedType": "boolean",
      "gormMap": {}
    },
    {
      "name": "role",
      "type": "one",
      "target": "RoleEntity",
      "validate": "required",
      "computedType": "RoleEntity",
      "gormMap": {}
    }
  ],
  "cliDescription": "Active invitations for non-users or already users to join an specific workspace"
}
public static Fields = {
  ...BaseEntity.Fields,
      coverLetter: 'coverLetter',
      targetUserLocale: 'targetUserLocale',
      value: 'value',
      workspace$: 'workspace',
      workspace: WorkspaceEntity.Fields,
      firstName: 'firstName',
      lastName: 'lastName',
      used: 'used',
          roleId: 'roleId',
      role$: 'role',
      role: RoleEntity.Fields,
}
}
